app/templates/components/public/ticket-list.hbs

```diff
@@ -88,12 +88,19 @@
                     @id={{concat ticket.id "_quantity"}}
                     @value={{ticket.orderQuantity}} />
                   <i class="dropdown icon"></i>
-                  <div class="default text">0</div>
+                  <div class="default text">{{if (eq this.tickets.length 1) 1 0}}</div>
```

Comment:
Have you verified that it actually selects 1 in code as well and not just in UI

Reply:
No it doesn't select 1, for that part I have added the below logic.

Reply:
just starting mapping from 1
```
{{#each (rannge 2 (add ticket.maxOrder 1)) as |count|}}
          <div class="item" data-value="{{map-value mapper count}}">{{count}}</div>
     {{/each}}
```